port_ocean/context/ocean.py

```diff
@@ -145,6 +146,21 @@ async def sync(
     async def sync_raw_all(self) -> None:
         await self.integration.sync_raw_all(trigger_type="manual")
 
+    def add_webhook_processor(
+        self,
+        path: str,
+        processor: type,
+        events_filter: Callable[[Any], bool] = lambda _: True,
+    ) -> None:
+        """
+        Registers a webhook processor for a specific path.
+
+        Args:
+            path: The path to register the webhook processor for.
+            processor: The processor to register.
+        """
```

Comment:
provide an example on how it will be used

Reply:
Added examples